### DISCUSSION TITLE
[Bug]

### DISCUSSION THREAD

**Discussion by TeD06910** [DanWBR/dwsim#690] [General]
**Describe the bug**
The Discharge temperature of a gas passing through a valve is clearly not calculated properly
By comparison with Hysys, DWSIM calculate much higher temp.

**Desktop (please complete the following information):**
 - OS: Windows 10

**Additional context**

**Comment by DanWBR** [comment]
@TeD06910 please provide more details or/and a flowsheet that can be reproduced before reporting such a serious thing (which is not true).

**Comment by TeD06910** [comment]
Hello Daniel,

First of all it is not serious and it is true (don’t appreciate too much your wording) or maybe I made a mistake in the configuration
Be sure I appreciate your software and I recognize the huge amount of work you did to make it accessible to any process engineer for free, for that many congratulation.

Apparently yes, there are some discrepancies on calculating the JT effect across the function : Valve, see big difference between Hysys and DWSIM
If I remember there was a post on same issue.

Regards,
Thierry

**Comment by DanWBR** [comment]
Again, if you do not provide a flowsheet to prove your generic affirmation that the valve doesn't work at all, I will delete this bug report.

**Comment by TeD06910** [comment]
In my email I have provided screenshot and file sources, did you get them ?

**Comment by DanWBR** [comment]
You must come to github and attach the zipped flowsheet

**Comment by TeD06910** [comment]
![DWSIM Test](https://github.com/user-attachments/assets/41354ff6-8e57-4c8a-bcc4-faf8625c81a1)
![Hysys Test](https://github.com/user-attachments/assets/914c3f60-3310-4552-b759-6e7e053ec589)

**Comment by TeD06910** [comment]
done

**Comment by TeD06910** [comment]
I see you have closed the ticket, does this means you are not going to check this valve calculation ?

**Comment by DanWBR** [comment]
@TeD06910 you closed it

**Comment by DanWBR** [comment]
@TeD06910 can you provide the composition of stream 10?

**Comment by TeD06910** [comment]
Please see the screenshot, you have the LP Stream property table showing the gas composition

**Comment by DanWBR** [comment]
@TeD06910 the composition of LP Stream and 10 (1-7) are different. Please provide 10 and 1-7 if possible

**Reply by TeD06910** [reply]
Hello
Compositions are the same, Hysis is in Percent and DWSIM in fraction
So you can use Hysys values and divide by 100 or use directly the values shown in DWSIM snapshot
